src/app/admin/students/services/students.service.ts

```diff
@@ -1,14 +1,45 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { GetStudentsInterface } from '../interfaces/get-students-interface';
+import { CreareUpdateStudentsInterface } from '../interfaces/creare-update-students-interface';
+import { ResponseInterface } from '../interfaces/response-interface';
 
 @Injectable()
 export class StudentsService {
 
+  private createStudentUrl = 'Student/insertData';
+  private readStudentUrl = 'Student/getStudentsByGroup';
+  private updateStudentUrl = 'Student/update';
+  private deleteStudentUrl = 'Student/del';
+  private getUserInfoURL = 'AdminUser/getRecords';
+
   private checkStudentNameURL = 'AdminUser/checkUserName';
   private checkStudentEmailURL = 'AdminUser/checkEmailAddress';
-  private checkGradebookID = 'Student/checkGradebookID';
 
-  constructor(private httpClient: HttpClient) { }
+  private checkGradebookIdURL = 'Student/checkGradebookID';
+
+  constructor(private httpClient: HttpClient) {}
+
+  getStudentsByGroup(id: number): Observable<GetStudentsInterface[] & ResponseInterface> {
+    return this.httpClient.get<GetStudentsInterface[] & ResponseInterface>(`${this.readStudentUrl}/${id}`);
+  }
+
+  deleteStudent(id: number): Observable<ResponseInterface> {
+    return this.httpClient.get<ResponseInterface>(`${this.deleteStudentUrl}/${id}`);
+  }
+
+  createStudent(body): Observable<CreareUpdateStudentsInterface|ResponseInterface> {
+    return this.httpClient.post<CreareUpdateStudentsInterface|ResponseInterface>(`${this.createStudentUrl}`, body);
+  }
+
+  updateStudent(id: number, body): Observable<CreareUpdateStudentsInterface|ResponseInterface> {
+    return this.httpClient.post<CreareUpdateStudentsInterface|ResponseInterface>(`${this.updateStudentUrl}/${id}`, body);
+  }
+
+  getUserInfo(id: string){
+    return this.httpClient.get(`${this.getUserInfoURL}/${id}`);
```

Comment:
Why do u need this lines of code if we already have same logic in shared service?

Reply:
Got it, will be refactored with using shared logic and tested everything in the next pull request today after merge of this one